store/structured/mongo/test/test.go

```diff
@@ -18,6 +19,9 @@ import (
 var database string
 var address string
 
+var cleanupHandlers []func()
+var cleanupHandlersMu sync.Mutex
```

Comment:
[nit] `cleanupHandlersMutex` so it is clear?

Reply:
I believe the -Mu suffix is well established as indicating a sync.Mutex.

If I grep for `'Mu\s+sync.Mutex'` I get 41 hits.

If I grep for `'Mutex\s+sync.Mutex'` I get 3 hits.

The numbers are similar when searching the Go stdlib code (40 and 8).